### PR TITLE
Add support for indexing split object without array even when solo

### PR DESCRIPTION
**Pull request by gregkohn**
https://github.com/riasvdv/craft-scout/issues/193

### CODE REVIEW

**Review by joshuapease — APPROVED** [review]
I'm stumped for a better name.

File: src/models/Settings.php [hunk 1]
```diff
@@ -42,6 +42,9 @@ class Settings extends Model
     /* @var int */
     public $batch_size = 1000;
 
+    /** @var bool */
+    public $useOriginalRecordIfSplitValueIsArrayOfOne = true;
```
Comment (davist11) [inline src/models/Settings.php:46]: May also be worth adding it to the config file:

https://github.com/vigetlabs/craft-scout/blob/master/config/scout.php

**Review by davist11 — APPROVED** [review]
Var name 🤷 

Could probably make it more succinct, but I think it's fine.

File: src/models/Settings.php (hunk 1 above)
Comment (gregkohn) [reply src/models/Settings.php:46]: good all, & a chance to add a better explanation

**Pull request merged**